### PR TITLE
Create bower.json

### PR DESCRIPTION
**Pull request by Den-dp**
related with https://github.com/web-animations/web-animations-js/pull/16

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "node_modules",
+    "templates",
+    "test",
+    ".travis.yml",
+    ".travis-setup.sh",
+    "Gruntfile.js",
+    "package.json",
+    "target-config.js",
+    "target-loader.js",
+    "web-animations.html",
```
Comment (dstockwell) [inline bower.json]: This one should not be ignored.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "keywords": [
+    "animations",
+    "polyfill"
+  ],
+  "license": "Apache-2.0",
+  "ignore": [
+    "node_modules",
+    "templates",
+    "test",
```
Comment (dstockwell) [inline bower.json]: Why not ignore the src directory, is it useful?

**Comment by dstockwell** [comment]
lgtm once `web-animations.html` is removed from the ignore list

**Comment by rjwright** [comment]
Landed as https://github.com/web-animations/web-animations-next/pull/296

**Pull request closed without merging**